ccd-definition/CaseField.json

```diff
@@ -181,5 +181,38 @@
     "HintText": "This is optional - the claimant will see your answer.",
     "FieldType": "TextArea",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "defendantSolicitor1claimResponseExtensionAccepted",
+    "Label": "Do you agree to extend the deadline to this date?",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "defendantSolicitor1claimResponseExtensionCounter",
+    "Label": "Would you like to set a different extended deadline?",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "defendantSolicitor1claimResponseExtensionCounterDate",
+    "Label": "New deadline",
+    "HintText": "This can't be later than 28 working days after the current deadline. All deadlines require a response before 4pm on the deadline day.",
+    "FieldType": "Date",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "defendantSolicitor1claimResponseExtensionResponse",
```

Comment:
@swalker125  - can we rename this to defendantSolicitor1claimResponseExtensionRejectionReason